.github/workflows/upload-coverage-report.yml

```diff
@@ -0,0 +1,53 @@
+name: Upload Coverage Report
+
+on:
+  workflow_run:
+    workflows:
+      - LLPC Docker CI
+    types:
+      - completed
+
+jobs:
+  retrieve-and-upload:
+    if: github.repository == 'GPUOpen-Drivers/llpc' && github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
+    name: "Configuration: ${{ matrix.config_tag }}"
+    runs-on: "ubuntu-20.04"
+    strategy:
+      fail-fast: false
+      matrix:
+        config_tag:  ["release_clang_coverage", "release_clang_shadercache_coverage_assertions"]
```

Comment:
Would it be possible to create just one comment? IE instead of running this job in 2 configurations, run one job but loop over all reports?

If this is tricky to implement, it's fine as-is.

Reply:
I thought about that but decided not to do it since there's no way to "loop" in GitHub Actions workflows, meaning some of the steps or workflow code would have to be manually repeated (for example the comment's contents or the download action). Since the job takes very little time, I prefer to leave as is until there's a more concrete need to change. This has the advantage that we can easily change the configurations in the matrix.